cmd/sync.go

```diff
@@ -313,8 +319,16 @@ func doSync(c *cli.Context) error {
 	go func() { _ = http.ListenAndServe(fmt.Sprintf("127.0.0.1:%d", config.HTTPPort), nil) }()
 
 	// Windows support `\` and `/` as its separator, Unix only use `/`
-	srcURL := strings.Replace(c.Args().Get(0), "\\", "/", -1)
-	dstURL := strings.Replace(c.Args().Get(1), "\\", "/", -1)
+	srcURL := c.Args().Get(0)
+	dstURL := c.Args().Get(1)
+	if runtime.GOOS == "windows" {
+		if !strings.Contains(srcURL, "://") && isFilePath(srcURL) {
+			srcURL = strings.Replace(srcURL, "\\", "/", -1)
+		}
+		if !strings.Contains(dstURL, "://") && isFilePath(dstURL) {
```

Comment:
```suggestion
		if !strings.Contains(dstURL, "://") {
```

Reply:
Hi, `strings.Contains(srcURL, "://")` can't cover this case `file:///C:a/b/c` ?